html/pages/index.html

```diff
@@ -1,4 +1,137 @@
----
-layout: default
-title: Home
----
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <!-- <meta content="" name=""> -->
+    <!-- <link href="" rel="stylesheet" type="text/css"> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+  </head>
+
+  <body>
+    <div class="wrapper">
+
+      <header>
+        <h1 class="logo">Movies&amp;Stuff</h1>
+        <nav class="navigation">
+          <ul>
+            <li> <a href="#movies"> Movies </a> </li>
+            <li> <a href="#tv-shows"> TV Shows </a> </li>
+            <li> <a href="#celebrities"> Celebs &amp; Photos</a> </li>
+            <li> <a href="#news"> News </a> </li>
+          </ul>
+        </nav>
+      </header>
+
+      <section class="suggested">
+          <!-- It seems that the following element must be a link to a video. It's just a plain image for now.
+          Background image will be set in css -->
+
+        <img src="{{root}}images/deadpool-preview.jpg" alt="deadpool-preview">
```

Comment:
We should place this image left. So we need some wrappers for the image and description on the right side.